src/components/extractFields.js

```diff
@@ -38,6 +38,14 @@ function getLon(properties) {
   }
 }
 
+function getBoundingBox(properties) {
+  if (properties['lbl:bbox']) {
```

Comment:
Upon further reflection, I now disagree (although not very strongly). The only differences between `if (object['prop'))` and `if (obect.hasOwnProperty('prop'))` are how it handles the number zero and empty strings. For bounding boxes, neither of those are valid values, so falling back to the standard bbox is fine.

For other fields it's mostly the same way. With population we explicitly want to ignore zero values and fall back, and while I mentioned the null island case (where we actually _do_ [incorrectly fall back to the `geom` lat/lon because the `lbl` lat/lon is 0, 0](https://whosonfirst.mapzen.com/spelunker/id/1/#18/-0.00004/-0.00013), it feels like the simplicity of the current syntax is worth the slight mishandling of that one case.

Am I missing an edge case that we are protected against with `hasOwnProperty`? I think I covered [all of them](https://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/)